### PR TITLE
Basic navbar

### PR DESCRIPTION
**Pull request by joshcwinton**
Description of PR here...

### Frontend:
- Created a basic navbar with components from bootstrap react
- created a temporary (?) logo
### Backend:
-

### CODE REVIEW

File: frontend/src/components/Chat.jsx [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
+  };
 
   render() {
     return (
       <div>
-        <ChatInput
-          onSubmitMessage={messageString => this.submitMessage(messageString)}
-        />
-        <ChatMessageList messages={this.state.messages}/>
+        <SHRDLUNavbar />
```
Comment (JasperCheung) [inline frontend/src/components/Chat.jsx]: Move this to `index.js` so it this shared across all our pages.

**Review by JasperCheung — CHANGES_REQUESTED** [review]
Move navbar to `index.js` so that it is shared between all our pages. Other than that everything looks good.
```
<Router>
  <navbar/>
  ....
 ```

**Review by JasperCheung — APPROVED** [review]
LGTM, merge whenever

**Comment by joshcwinton** [comment]
💸 💸 💸 💸

**Pull request merged**